src/components/lowering_replay.js

```diff
@@ -42,6 +42,7 @@ class LoweringReplay extends Component {
       state: null,
       cruise: props.cruise,
       lowering: props.lowering,
+      utcScrubber: true,  // add a toggle in the UI or possibly put this in a user setting
```

Comment:
Maybe a better name for this is something like `useRelativeTimestamp` or something. I think the operative bit is that it's absolute vs relative time rather than the timezone. Not a big deal.

I recommend using `TODO` for comments of this nature, so that it's easy to search the codebase for them.

Reply:
Done!